src/main/js/core/Shape/shapeDefinitions.js

```diff
@@ -271,7 +271,7 @@ export const TRIANGLE_DOWN_LIGHT = {
         { fill: "#FFF", d: "M42.7 7L24 38.2 5.3 7z" },
         { d: "M0 4l24 40L48 4H0zm42.7 3L24 38.2 5.3 7h37.4z" }
     ],
-    options: { x: -6, y: -6, scale: 0.25 }
+    options: { x: -6.2, y: -6, scale: 0.25 }
```

Comment:
Why this triangle_down_light is little different than triangle_light? I mean why both have different values?

Reply:
They no particular reason for why those are not same. I just tried to keep those shapes as perfect as i can. In that process i did adjusted in this way, so that they will look good on the reference line as well as dateline.